server/api/elections.py

```diff
@@ -107,3 +107,31 @@ def delete_election(election: Election):
 
     db_session.commit()
     return jsonify(status="ok")
+
+
+@api.route("/organizations", methods=["GET"])
+def list_organizations_and_elections():
+    user_type, user_key = get_loggedin_user(session)
+    if user_type != UserType.AUDIT_ADMIN:
+        raise Forbidden(f"Access forbidden for user type {user_type}")
```

Comment:
Yeah, there is a decorator for this for most endpoints. Two reasons I didn't use it:
- It expects that all endpoints have an election_id in the url
- I would still need to have some code to verify that the audit admin id in the url matches the currently logged in user (based on your recommendation above)

I could change the decorator to do this, or add a new decorator, but since this endpoint is the only one like this, I'd rather wait for a pattern to emerge before abstracting.